src/client/pages/OfferNew/Introduction/Sidebar/DetailsModal/Details/SwedishDetails.tsx

```diff
@@ -0,0 +1,245 @@
+import React from 'react'
+import { FieldArray, FormikProps } from 'formik'
+import {
+  InputGroup,
+  InputGroupDeleteButton,
+  InputGroupRow,
+} from 'components/inputs'
+import { ExtraBuildingInput, ExtraBuildingType } from 'data/graphql'
+import { EditQuoteInput } from 'data/graphql'
```

Comment:
i just saw this, how does it pass eslint? We don't have a rule for multiple imports from the same file?  🤨